testing/internal/e2e/infra/docker.go

```diff
@@ -118,12 +119,16 @@ func GetDbInitInfoFromContainer(t testing.TB, pool *dockertest.Pool, container *
 	err = json.Unmarshal(buf.Bytes(), &dbInitInfo)
 	require.NoError(t, err, buf.String())
 
+	os.Setenv("E2E_PASSWORD_AUTH_METHOD_ID", dbInitInfo.AuthMethod.AuthMethodId)
+	os.Setenv("E2E_PASSWORD_ADMIN_LOGIN_NAME", dbInitInfo.AuthMethod.LoginName)
+	os.Setenv("E2E_PASSWORD_ADMIN_PASSWORD", dbInitInfo.AuthMethod.Password)
```

Comment:
I'm confused by this change. Why is this being set here? If we need to set these values, I would expect that to be done in the function that's calling whatever it is that needs these in the environment. Do we need to do this at all though?

Reply:
I was originally thinking that creating the docker image, you may want to have these automatically set, but I was going back and forth on it. I'll move it back for clarity.

We do need to set some of these environment variables because some of the helper methods needs them (like enos setting up infrastructure and setting environment variables, these methods will need to do the same). I'll look to do some refactoring to see if we can reduce some of it.